packages/drivers/driver-web-cache/src/FluidCache.ts

```diff
@@ -47,6 +49,17 @@ export interface FluidCacheConfig {
 	 * If an entry exists in the cache, but is older than this value, the cached value will not be returned.
 	 */
 	maxCacheItemAge: number;
+
+	/**
+	 * To improve perf, if this property is set as false, then db will not be closed immediately after usage.
+	 * Default is true.
+	 */
+	immediateClose?: boolean;
+
+	/**
+	 * Each time db is opened, it will remain open for this much time. If not provided, default value will be 2s.
+	 */
+	closeDbAfter?: number;
```

Comment:
I think you need to mention in comments, or in name of property what number means - is it in seconds, milliseconds, or something else?